daml.yaml

```diff
@@ -3,7 +3,7 @@
 
 sdk-version: 1.4.0
 name: dabl-integration-exberry
-version: 1.0.0
+version: 0.8.0
```

Comment:
Change dar version to match the minor version of the `dit-meta.yaml` file

Reply:
The DAR version can and should be managed independently of the DIT version. Toward the beginning of `ddit`, we used to enforce the the DAR file had the same version as the DIT file - this causes problems when the DIT needs to change, but the DAR can stay static. (Which is very often the case.)

In light of that, I.suggest you bump the DAR version  up to `1.0.1` or something to keep the version number increasing.